lib/lte_link_control/lte_lc.c

```diff
@@ -862,7 +862,7 @@ int lte_lc_psm_get(int *tau, int *active_time)
 	char active_time_str[9] = {0};
 	char tau_ext_str[9] = {0};
 	char tau_legacy_str[9] = {0};
-	char response[160] = { 0 };
+	static char response[160] = { 0 };
```

Comment:
There are a couple of other functions which also have 160 byte (4 * 40) buffers in stack. It can be that those functions are otherwise using less stack and are not a problem, but this anyway makes me wonder whether this buffer could be global instead. At least we should prevent having static buffers in several functions. :)

Reply:
Yeah, this is more of a quick fix to address the fact that AT notifications are now received in the system workqueue context using AT monitor as opposed to at_cmd thread context when we used at_notif.
I created CIA-541 to look more into optimizing buffer allocations in the link controller after the release.